### PR TITLE
[eclipse/xtext#1573] Ensure that all plugins do have a ManifestBuilder

### PR DESCRIPTION
**Pull request by szarnekow**
[eclipse/xtext#1573] Ensure that all plugins do have a ManifestBuilder

### CODE REVIEW

File: gradle/eclipse-project-layout.gradle [hunk 1]
```diff
@@ -4,6 +4,7 @@
  */
 
 def isTestProject = name.endsWith('tests')
+def isPlugin = new File("$projectDir/META-INF/MANIFEST.MF").exists()
```
Comment (cdietrich) [inline gradle/eclipse-project-layout.gradle]: am not sure if this is the common pattern or `project.file("META-INF/MANIFEST.MF")`

File: gradle/eclipse-project-layout.gradle [hunk 2]
```diff
@@ -73,5 +74,10 @@ eclipse {
 	project {
 		natures 'org.eclipse.xtext.ui.shared.xtextNature'
 		buildCommands.add(0,new org.gradle.plugins.ide.eclipse.model.BuildCommand('org.eclipse.xtext.ui.shared.xtextBuilder'))
+		if (isPlugin) {
```
Comment (cdietrich) [inline gradle/eclipse-project-layout.gradle]: nature not needed?

Comment (szarnekow) [reply gradle/eclipse-project-layout.gradle]: It's added in the line below

File: gradle/eclipse-project-layout.gradle (hunk 1 above)
Comment (szarnekow) [reply gradle/eclipse-project-layout.gradle]: I don't know either. Should I change this?

Comment (cdietrich) [reply gradle/eclipse-project-layout.gradle]: i think we can keep it

**Pull request merged**